### DISCUSSION TITLE
Store color and brightness level on the ESP

### DISCUSSION THREAD

**Discussion by raybantracing** [sblantipodi/firefly_luciferin#89] [Q&A] [answered]
Hi,
is it possible to store last color used and last brightness level used on the ESP?

I ask this because I use Luciferin as a LED lamp when my PC is off.
If the ESP reboots, it forgets the last color and brightness used.

**Comment by sblantipodi** [comment, marked as answer]
Hi @raybantracing,
thanks for using Luciferin.

It is not possible currently, we have a feature request here:
https://github.com/sblantipodi/glow_worm_luciferin/issues/38

it will be implemented soon.

**Comment by sblantipodi** [comment]
closing since the feature has been implemented
